packages/flutter/lib/src/cassowary/constant_member.dart

```diff
@@ -6,6 +6,8 @@ import 'expression.dart';
 import 'equation_member.dart';
 import 'term.dart';
 
+/// A member of a constraint expression that represent a constant at the time
+/// the constraint is added to the solver.
```

Comment:
general note throughout: add more hyperlinks. For example, `[Solver]` rather than `solver`, `constraint [Expression]` rather than `constraint expression`, etc.

The idea is to make it possible for people to follow breadcrumbs from anywhere to anywhere, falling down "wikipedia holes" in our docs.